.github/workflows/main.yml

```diff
@@ -116,6 +116,19 @@ jobs:
           echo '${{ steps.generate_env_file.outputs.env_file }}' >> $APP_ENV_PATH
           cat $APP_ENV_PATH
 
+      - name: Deploy to Transifex
+        working-directory: client
+        if: ${{ steps.applicable_check.outputs.flag }}
+        id: deploy_transifex
+        env:
+          TRANSIFEX_TOKEN: ${{ secrets[format('{0}_CLIENT_ENV_NEXT_PUBLIC_TRANSIFEX_TOKEN', env.ENVIRONMENT)] }}
+          TRANSIFEX_SECRET: ${{ secrets[format('{0}_CLIENT_ENV_NEXT_PUBLIC_TRANSIFEX_SECRET', env.ENVIRONMENT)] }}
```

Comment:
@barbara-chaves I realised that none of those two variables should start with `NEXT_PUBLIC_` in its name:

- `STAGING_CLIENT_ENV_NEXT_PUBLIC_TRANSIFEX_TOKEN` is only used server side.
- `STAGING_CLIENT_ENV_NEXT_PUBLIC_TRANSIFEX_SECRET` is only used during the deployment process.

I think both variables should be renamed as follows:

- `CLIENT_ENV_TRANSIFEX_TOKEN` (should be available on all environments)
- `STAGING_CLIENT_ENV_TRANSIFEX_SECRET`

Make sure to update the `README.md` file as well.

Reply:
Ok!